Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -298,8 +304,12 @@ final class SyncChangeStreamTests: MongoSwiftTestCase {
     /// Prose test 1 of change stream spec.
     /// "ChangeStream must continuously track the last seen resumeToken"
     func testChangeStreamTracksResumeToken() throws {
-        guard MongoSwiftTestCase.topologyType != .single else {
-            print(unsupportedTopologyMessage(testName: self.name))
+        let testRequirements = TestRequirement(
+            acceptableTopologies: ["replicaSet", "sharded"].map { TestTopologyConfiguration(from: $0) })
```

Comment:
Rather than mapping from strings, this can just be `[.replicaSet, .sharded]`, using the enum cases directly.

Reply:
nice :)